src/modules/75-cf/pages/onboarding/OnboardingPage.tsx

```diff
@@ -61,14 +68,13 @@ export const OnboardingPage = (): React.ReactElement => {
                   history.push(routes.toCFOnboardingDetail({ accountId, orgIdentifier, projectIdentifier }))
                 }}
               />
-            </Layout.Horizontal>
-          </Layout.Vertical>
+            </Layout.Vertical>
+          </Container>
+
           {/* Right side - img */}
-          <Layout.Horizontal>
-            <img src={ffOnboarding} width={380} height={320} />
-          </Layout.Horizontal>
+          <img src={ffOnboarding} width={380} height={320} />
```

Comment:
All images should have `alt` attributes. If the image is purely for decoration then it should have an empty `alt` attribute.
```suggestion
          <img src={ffOnboarding} width={380} height={320} alt="" />
```

Reply:
Have added